google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -592,4 +592,17 @@
     <className>com/google/cloud/spanner/AsyncTransactionManager$CommitTimestampFuture</className>
     <method>java.lang.Object get()</method>
   </difference>
+
```

Comment:
Clirr does not support Java 8 features, so it thinks these are breaking changes, where in fact we have provided default implementations for the added interface methods. No need for a major release here.